repos/system_upgrade/el7toel8/models/kernelcmdlineargs.py

```diff
@@ -4,9 +4,20 @@
 
 class KernelCmdlineArg(Model):
     """
-    Single kernel command line argument to include to RHEL-8 kernel cmdline
+    Single kernel command line parameter with/without a value
+
+    When produced alone, represents a parameter to include in RHEL-8 kernel cmdline.
     """
```

Comment:
We will have to change that later, as this looks very confusing. I would tell, we should use different model completely to represent cmdline parameters that should be included into the RHEL-8 kernel cmdline. @oamg/developers it's really wrong to have two different usecases for one model. but the rest is ok. merging.